### PR TITLE
doc: update versions for v1.4.99-dev1

### PR DESCRIPTION
**Pull request by ru-fu**
Update the version information for dev tag v1.4.99-dev1

Ref. NSCDK-7101

Signed-off-by: Ruth Fuchss <ruth.fuchss@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -216,11 +216,11 @@
 
 .. _`J-Link Software and Documentation Pack`: https://www.segger.com/downloads/jlink
 
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x64
-.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_macos
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x64
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v520x_win_x64.zip
```
Comment (ru-fu) [inline doc/nrf/links.txt:219]: These need to be updated once we know the version. @tejlmand

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -228,11 +228,11 @@
 
 .. _`J-Link Software and Documentation Pack`: https://www.segger.com/downloads/jlink
 
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x64
-.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_macos
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x64
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x64.zip
```
Comment (ru-fu) [inline doc/nrf/links.txt:231]: @tejlmand Are these the correct links?

Comment (tejlmand) [reply doc/nrf/links.txt:231]: they should be, but still waiting for final upload to happen.

File: doc/nrf/links.txt [hunk 3]
```diff
@@ -228,11 +228,11 @@
...
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x86.zip
+.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_linux_x64.tar.gz
```
Comment (tejlmand) [inline doc/nrf/links.txt:233]: This is wrong, the text describes mac, but link is Linux:
```suggestion
.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_macos_x64.dmg
```

File: doc/nrf/links.txt [hunk 4]
```diff
@@ -228,11 +228,11 @@
...
+.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_linux_x86.tar.gz
+.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_macos_x64.dmg
```
Comment (tejlmand) [inline doc/nrf/links.txt:235]: Link is wrong, should be pointing to the Linux version:
```suggestion
.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_linux_x64.tar.gz
```

File: doc/nrf/links.txt [hunk 5]
```diff
@@ -228,11 +228,11 @@
 
 .. _`J-Link Software and Documentation Pack`: https://www.segger.com/downloads/jlink
 
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x64
-.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_macos
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x64
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x64.zip
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x86.zip
```
Comment (tejlmand) [inline doc/nrf/links.txt:232]: Link is wrong, according to the text this should point to 64bit version:
```suggestion
.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x64.zip
```

File: doc/nrf/links.txt (hunk 2 above)
Comment (tejlmand) [reply doc/nrf/links.txt:231]: actually., the link points to the 64bit version, not the 32bit as text describes.
So should be:
```suggestion
.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v530a_win_x86.zip
```

**Review by tejlmand — CHANGES_REQUESTED** [review]
Links has been moved around.

They are correct, but placed with wrong description.

File: doc/nrf/links.txt (hunk 2 above)
Comment (ru-fu) [reply doc/nrf/links.txt]: Gosh, how did I manage to do that ...?!
Fixed now. Thank you, good catch!

**Comment by ru-fu** [comment]
Screenshot for the welcome page:
image

Release notes page:
image

**Comment by tejlmand** [comment]
> 
> 
> Screenshot for the welcome page:

I notice the text states: *You are looking at the documentation for a development **release**.*

Technically this is not a *release*, so I would suggest we change this to the term:
*You are looking at the documentation for a development **tag**.*

as that is what we are actually doing. @carlescufi  @shantha-14 any thoughts on this ?

File: doc/nrf/index.rst [hunk 6]
```diff
@@ -3,6 +3,13 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v1.4.99-dev1 is a development release and will be replaced by v1.5.0 in the future.
```
Comment (tejlmand) [inline doc/nrf/index.rst:8]: I think the term `development tag` is more correct, and will give the reader a more correct interpretation of the tag, compared to a release.

```suggestion
   |NCS| v1.4.99-dev1 is a development tag and will be replaced by v1.5.0 in the future.
```

**Comment by shantha-14** [comment]
> > Screenshot for the welcome page:
> 
> I notice the text states: _You are looking at the documentation for a development **release**._
> 
> Technically this is not a _release_, so I would suggest we change this to the term:
> _You are looking at the documentation for a development **tag**._
> 
> as that is what we are actually doing. @carlescufi @shantha-14 any thoughts on

**Comment by shantha-14** [comment]
yes I agree, it should be development tag @ru-fu  and @tejlmand

**Review by tejlmand — APPROVED** [review]
lgtm

**Comment by ru-fu** [comment]
Updated the note that details production/development support:
image

@jdstapes @shantha-14

**Comment by b-gent** [comment]
Checked the output and everything looks correct here.

**Review by b-gent — APPROVED** [review]

File: doc/nrf/index.rst [hunk 7]
```diff
@@ -3,14 +3,25 @@
...
 
 .. note::
-   nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth Mesh protocols are supported for development in |release| for prototyping and evaluation.
-   Support for production and deployment in end products is coming soon.
+
+   |NCS| v1.4.99-dev1 supports prototyping and evaluation, but should not be used for production and deployment in end products.
+
+.. previous note about development and production support
+..   nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth Mesh protocols are supported for development in |release| for prototyping and evaluation.
+..   Support for production and deployment in end products is coming soon.
```
Comment (greg-fer) [inline doc/nrf/index.rst:24]: Nice, good that we keep it for later like this.

**Review by greg-fer — APPROVED** [review]

**Comment by tejlmand** [comment]
Just discovered, should we update: 
https://github.com/nrfconnect/sdk-nrf/blob/3a544e665dc458a10a8e2f1777b9c7e23342964e/VERSION#L1

to contain the tag ?
@carlescufi thoughts ?

**Review by ioannisg — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**